defi/Element/readme.md

```diff
@@ -0,0 +1,27 @@
+# Element Protocol
+## 原理
+利率浮動是永續借貸資金池的必然現象。
+固定利率有一下几种形式：
+
+1. 零息债券（Zero coupon bond）：
+不支付利息的債券，但通常他的交易价格会低于面值，到期後會按面值支付給債券持有人
+2.  利息重分配
+基于既有的利息來源，可以是浮動利率的存款，也可以是 Yield Farming，衍生出一個交易利率的次級市場。而根據交易模式不同，又可以再分成本金-利息拆分与结构型商品两种。
+ - 本金-利息分離
+ 給定一個收益來源，我們可以將投資的回報拆分成本金和利息兩部分，並分別定價。
+ 如：你也可以理解成 A 和 B 兩人分別出 $9,600 和 $400 一起投資到 Aave，並約好一年後 A 拿 $10,000 而 B 拿到剩下的部分，達到事先約定利潤分配的效果。
+ Element Finance就是以上思路
+- 结构型商品 (Structured Product)
+  既然未来利息是不确定的，那根据每个人的分享偏好和资金承受能力，根据个人需求去分摊风险；
+  结构型基金可針對投資人對市場預期的不同，將利息收益拆分成不同等級，並將其重新組合成不同的金融衍生品。
+  如：设计一个以5%为界的双层架构，A类产品风险较低（固定利率），B类产品风险较高但高收益产品（浮动利率）， 基金的錢全部都拿去 Aave放贷，若利息超过5%，B类投资人可以拿超额收益，
+  如果不足5%，则B类投资人的钱会拿去填补不足。
+  相关协议：BarnBridge, Tranche
+
+## 参考链接
+
+
+
+
+
+
```

Comment:
why this part is Traditional Chinese 😂

Reply:
Need somebody translate this later...I think Chinese is OK for now.